src/platforms/blackpillv2/platform.c

```diff
@@ -149,6 +149,22 @@ void platform_target_set_power(const bool power)
 {
 	gpio_set_val(PWR_BR_PORT, PWR_BR_PIN, !power);
 }
+
+/**
```

Comment:
Please do not use Doxygen style comments - a simple block comment more than suffices. For example:
```c
/*
 * A dummy implementation of platform_target_voltage_sense as
 * the Blackpillv2 has no ability to sense the voltage on the power pin.
 * This function is only needed for implementations that allow the target
 * to be powered from the debug probe.
 */
```

Reply:
Thanks for your reply! The change to a block comment is implemented in d32b652.